libindy_vdr/src/common/error.rs

```diff
@@ -130,6 +130,12 @@ impl From<zmq::Error> for VdrError {
     }
 }
 
+impl From<sled::Error> for VdrError {
```

Comment:
Can leave this as-is for now, but I think we can use `#[from(sled::Error)]` from `thiserror` and it will derive it properly.